timeseries/src/autogluon/timeseries/learner.py

```diff
@@ -106,6 +136,50 @@ def _fit(
 
         self._time_fit_training = time.time() - time_start
 
+    def _preprocess_static_features(
+        self,
+        train_data: TimeSeriesDataFrame,
+        static_feature_metadata: FeatureMetadata,
+        val_data: Optional[TimeSeriesDataFrame] = None,
+    ) -> Tuple[TimeSeriesDataFrame, Optional[TimeSeriesDataFrame]]:
+        self.feature_pipeline = ContinuousAndCategoricalFeatureGenerator(feature_metadata=static_feature_metadata)
+
+        # Avoid modifying train_data outside of predictor
+        train_data = train_data.copy(deep=False)
```

Comment:
is this working as intended? should we maybe cover it with tests in `TimeSeriesDataFrame`?